pykwasm/src/pykwasm/call.py

```diff
@@ -107,6 +107,9 @@ def run_method(w3, contract, sender, eth, method, params):
             msg = f'Node RPC encountered an error: {e.message}'
         print(msg, file=sys.stderr)
         sys.exit(1)
+    except ContractLogicError as e:
+        print(f'Contract logic error: {e.message}', file=sys.stderr)
+        sys.exit(1)
```

Comment:
This is the right logic, but I have a slight preference to put it in the if/then/else statement above since it shares the same logic by:

1. adding the `ContractLogicError` to the tuple of caught errors and;
2. adding a new case to the if/then/else statement for printing its message